pyerrors/input/sfcf.py

```diff
@@ -6,17 +6,41 @@
 import re
 import numpy as np  # Thinly-wrapped numpy
 from ..obs import Obs
+from . import utils
 
-
-def read_sfcf(path, prefix, name, **kwargs):
-    """Read sfcf C format from given folder structure.
+def read_sfcf(path, prefix, name, quarks='.*', noffset=0, wf=0, wf2=0, **kwargs):
+    """Read sfcf c format from given folder structure.
 
     Parameters
     ----------
-    im -- if True, read imaginary instead of real part of the correlation function.
-    single -- if True, read a boundary-to-boundary correlation function with a single value
-    b2b -- if True, read a time-dependent boundary-to-boundary correlation function
-    names -- Alternative labeling for replicas/ensembles. Has to have the appropriate length
+    quarks: str
+        Label of the quarks used in the sfcf input file. e.g. "quark quark"
+        for version 0.0 this does NOT need to be given with the typical " - " that is present in the output file,
+        this is done automatically for this version
+    noffset: int
+        Offset of the source (only relevant when wavefunctions are used)
+    wf: int
+        ID of wave function
+    wf2: int
+        ID of the second wavefunction (only relevant for boundary-to-boundary correlation functions)
+    im: bool
+        if True, read imaginary instead of real part of the correlation function.
+    b2b: bool
+        if True, read a time-dependent boundary-to-boundary correlation function
+    single: bool
+        if True, read time independent boundary to boundary correlation function
+    names: list
+        Alternative labeling for replicas/ensembles. Has to have the appropriate length
+    ens_name : str
+        replaces the name of the ensemble
+    version: str
+        version of SFCF, with which the measurement was done. if the compact output option (-c) was spectified, append a c to the version (e.g. "1.0c")
```

Comment:
Ideally, one could also specify `a` for the append mode.

Reply:
Yes, this was my idea as well. When starting to implement that feature, I will go with this parameter